### PR TITLE
minor menu clean-up

### PR DESCRIPTION
**Pull request by fabiangreffrath**

### CODE REVIEW

File: src/mn_setup.c [hunk 1]
```diff
@@ -3246,8 +3244,8 @@ static setup_menu_t gen_settings5[] = {
     {"Sprite Translucency", S_ONOFF | S_STRICT, OFF_CNTR_X, M_SPC,
      {"translucency"}},
 
-    {"Translucency Filter", S_NUM | S_ACTION | S_PCT, OFF_CNTR_X, M_SPC,
-     {"tran_filter_pct"}, .action = MN_Trans},
+    {"Invulnerability effect", S_CHOICE | S_STRICT, OFF_CNTR_X, M_SPC,
+     {"invul_mode"}, .strings_id = str_invul_mode, .action = R_InvulMode},
```
Comment (rfomin) [inline src/mn_setup.c:3248]: Why move this, but not "Screen Flashes" or "Screen Wipe Effect"? I like to keep "Invulnerability Effect" in the "Misc" tab.

Comment (fabiangreffrath) [reply src/mn_setup.c:3248]: Both partial invisibility and invulnerability are in-game rendering effects. Think of, they can both be reached by the IDBEHOLD cheat.

Comment (rfomin) [reply src/mn_setup.c:3248]: "Screen Flashes" also in-game rendering effect and "Berserk" can be reached with IDBEHOLD 😄

**Review by rfomin — APPROVED** [review]
Thanks!

**Pull request merged**